x-pack/plugins/data_usage/server/plugin.ts

```diff
@@ -53,6 +53,8 @@ export class DataUsagePlugin
   }
   setup(coreSetup: CoreSetup, pluginsSetup: DataUsageSetupDependencies): DataUsageServerSetup {
     this.logger.debug('data usage plugin setup');
+    const dataUsageService = new DataUsageService(this.dataUsageContext);
```

Comment:
I was thinking of adding only the auto ops service to the appContextService on start, but this is much neater. 🚀